spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  describe '#DMV data services' do
+    it 'creates vehicles' do
+      factory = VehicleFactory.new
+      wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+    
+      expect(factory.create_vehicles(wa_ev_registrations)[0]).to be_instance_of Vehicle
+      factory.create_vehicles(wa_ev_registrations).each do |data|
+      expect(data).to be_instance_of Vehicle
```

Comment:
Like I mentioned in a similar test, I think you could also use this block to assert some things about each vehicle object's attributes.